pdal/Geometry.cpp

```diff
@@ -74,6 +74,19 @@ Geometry::Geometry(OGRGeometryH g, const SpatialReference& srs) :
 }
 
 
+Geometry::Geometry(OGRGeometryH gh, OGRSpatialReferenceH srs) :
```

Comment:
The way to do this, IMO, is to add a ctor to GDALUtils::SpatialRef that takes an OGRSpatialReferenceH and clones it.  Then everything else stays the same.  No changes to Geometry or Polygon or SpatialReference and it keeps the GDAL code isolated.